src/pages/landing/[landingPage]/index.tsx

```diff
@@ -32,14 +32,42 @@ type DocumentsType = {
 const LandingPage = ({
   landingPage,
 }: InferGetServerSidePropsType<typeof getServerSideProps>) => {
-  const { pageTitle, pageDescription, documents, collections, articles } =
-    landingPage
+  const {
+    pageTitle,
+    badge,
+    hero,
+    pageDescription,
+    documents,
+    collections,
+    articles,
+  } = landingPage
+
+  const badgeImage = badge?.url.length ? badge.url : '/img/default_badge.png'
+  const heroImage = hero?.url.length ? hero.url : ''
 
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const pageContent = (): any => {
     return (
       <>
-        <h1 className={styles.pageTitle}>{pageTitle}</h1>
+        <div className={styles.pageTitleContainer}>
```

Comment:
Thought(non-blocking): I wonder if we should refactor this so that most of `pageContent` function is in a separate component so that we can display it in storybook. I don't think this needs to happen now though since the page is already not in storybook. Probably we should come to some sort of consensus on this type of thing, maybe next design eng sync.